docs/source/aasequences.rst

```diff
@@ -90,6 +90,37 @@ Which will print
     Glutamate : 147.05315928710002
     Arginine : 174.1116764466
 
+The peptide N- and C-Terminus as well as the residues themself can be modified.
```

Comment:
Should that be "protein" instead of "peptide"?

Reply:
hmm I think it might be both. Maybe just write "The N- and C-Terminus ..."